package Exception;

public class S1 {

	public static void main(String[] args) {
		int[] test;
		test = new int[5];
		
		System.out.println("Assign a value to test[10]");
		
		test[10] = 80; 
		System.out.println("80 has been assigned to test[10]");
		System.out.println("Successfully executed");

	}

}
